scripts/index.js

```diff
@@ -57,8 +65,14 @@ const resolvedBuildPath = buildPath ? handleBuildPath(buildPath) : paths.appBuil
 // update the paths in config
 config.output.path = resolvedBuildPath;
 config.output.publicPath = publicPath || '';
-config.output.filename = `js/bundle.js`;
-config.output.chunkFilename = `js/[name].chunk.js`;
+
+// Grab output names from cli args, otherwise use some default naming.
+const fileNameToUse = outputFilename ? `${outputFilename}` : `js/bundle.js`;
+const chunkNameToUse = chunkFilename ? `${chunkFilename}` : `js/[name].chunk.js`;
+// If cli user adds .js, respect that, otherwise we add it ourself
+config.output.filename = fileNameToUse.slice(-3) != '.js' ? fileNameToUse + '.js' : fileNameToUse;
```

Comment:
Personally I prefer to use template string rather than string concatenation.

```suggestion
config.output.filename = fileNameToUse.slice(-3) !== '.js' ? `${fileNameToUse}.js` : fileNameToUse;
```
Same applies for chunkName.